common/src/main/java/com/wynntils/wc/utils/WynnItemMatchers.java

```diff
@@ -29,6 +33,29 @@ public static boolean isHealingPotion(ItemStack stack) {
                                 .contains(ChatFormatting.RED + "Potion of Healing"));
     }
 
+    public static boolean isCraftedHealingPotion(ItemStack stack) {
+        System.out.println(stack.getHoverName());
```

Comment:
Probably remove these 2 debugs

Reply:
A later thing to do is to have a general purpose consumable parser

Reply:
> A later thing to do is to have a general purpose consumable parser

Will add it in this PR and it could also tighten the crafted potion checks.